src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs

```diff
@@ -1,60 +1,68 @@
-﻿using GroupDocs.Annotation.Domain;
-using GroupDocs.Annotation.Domain.Containers;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
+using System.Collections.Generic;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class TexStrikeoutAnnotator : AbstractSvgAnnotator
+    public class TexStrikeoutAnnotator : BaseAnnotator
     {
-        public TexStrikeoutAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
+        private StrikeoutAnnotation strikeoutAnnotation;
+
+        public TexStrikeoutAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
+            strikeoutAnnotation = new StrikeoutAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)